soroban-settings/phase1.json

```diff
@@ -295,8 +295,8 @@
     {
       "state_archival": {
           "max_entry_ttl": 3110400,
-          "min_temporary_ttl": 16,
-          "min_persistent_ttl": 120960,
+          "min_temporary_ttl": 17000,
+          "min_persistent_ttl": 2073600,
           "persistent_rent_rate_denominator": 535680,
           "temp_rent_rate_denominator": 5356800,
           "max_entries_to_archive": 100,
```

Comment:
With this new minimum TTL settings, we can update `starting_eviction_scan_level` to 7 since it is no longer possible for entries to expire in level 6.

Reply:
If we reduce one of the ttl values below 16384, we should reduce the scan size as well right? Do you foresee any issues with this?

Reply:
Correct, but this should be fine. We have a test that reduces the scan level and we've done this before on future net without issue.

Reply:
Done